Calculator/Calculator/Models/ExpressionParser.swift

```diff
@@ -7,30 +7,31 @@
 
 enum ExpressionParser {
     static func parse(from input: String) -> Formula {
-        var formula = Formula(operands: CalculatorItemQueue<Double>(), operators: CalculatorItemQueue<Operator>())
-        let componentsArray = componentsByOperators(from: input)
+        var operatorsQueue = CalculatorItemQueue<Operator>()
+        var operandsQueue = CalculatorItemQueue<Double>()
+        let operandsArray = componentsByOperators(from: input)
         
-        componentsArray.forEach{ component in
-            if let operand = Double(component) {
-                formula.operands.enqueue(operand)
-            } else if let operatorSymbol = Operator(rawValue: Character(component)) {
-                formula.operators.enqueue(operatorSymbol)
-            }
+        operandsArray.compactMap { Double($0) }.forEach { component in
+            operandsQueue.enqueue(component)
+        }
+        
+        input.compactMap { Operator(rawValue: $0) }.forEach { component in
+            operatorsQueue.enqueue(component)
```

Comment:
고차함수 활용 👍👍👍
고차함수를 체이닝할 때 개행을 추가해주면 좀 더 가독성이 좋아지는 것 같아요.
단축인자이름도 컨벤션 중 하나라고 생각하기 때문에 가능하다면 맞춰 주시는 것이 좋다고 생각합니다. 😊
이 부분은 개인 취향과 팀에서 정하는 룰대로 맞춰가는 것이니 기준을 잘 정해두시면 좋습니다~!
참고만 해주세요!
```suggestion
        operandsArray.compactMap { Double($0) }
            .forEach { operandsQueue.enqueue($0) }
        
        input.compactMap { Operator(rawValue: $0) }
            .forEach { operatorsQueue.enqueue($0) }
```